.changeset/cli-args-children-example.md

```diff
@@ -0,0 +1,5 @@
+---
+"@zazuko/cube-hierarchy-query": patch
+---
+
+Add CLI arguments to "children" example to specify cube and dimension
```

Comment:
I'm not sure we actually need a changeset. As in, do modifications in the examples warrant publishing a new package version?